run-cmake-format.py

```diff
@@ -32,6 +32,7 @@
     'cpp/cmake_modules/FindBrotli.cmake',
     'cpp/cmake_modules/FindClangTools.cmake',
     'cpp/cmake_modules/FindDoubleConversion.cmake',
+    'cpp/cmake_modules/FindFlatbuffersAlt.cmake',
```

Comment:
I suppose `FindFlatbuffersAlt.cmake` is missing from the PR?

Reply:
Fixed